openstack/glance/templates/etc/_glance-policy.json.tpl

```diff
@@ -9,6 +9,7 @@
     "get_images": "",
     "modify_image": "",
     "publicize_image": "rule:context_is_cloud_admin",
+    "communitize_image": "role:image_admin",
```

Comment:
please add the permission for cloud_image_admin, e.g. `role:image_admin or rule:context_is_admin`